app/controllers/api/cloud_volumes_controller.rb

```diff
@@ -1,6 +1,8 @@
 module Api
   class CloudVolumesController < BaseController
     include Subcollections::Tags
+    include Subcollections::Snapshots
```

Comment:
again, we have a model called Snapshots, which is different from CloudVolumeSnapshot.

While I think it would be great if we could have '/snapshot' as a url, I really don't want us to call this subcollection snapshots since this is the wrong name/will be confusing if we want vms/snapshots

Reply:
Already posted on the [issue](https://github.com/ManageIQ/manageiq-api/issues/1094) raised.